jenkins/bp/ec2-py27-blueprint.yaml

```diff
@@ -0,0 +1,196 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+description: >
+  This blueprint deploy EC2 for integration tests
+imports:
+  - http://cloudify.co/spec/cloudify/5.0.5/types.yaml
+  - plugin:cloudify-aws-plugin
+  - plugin:cloudify-utilities-plugin?version= >=1.22.1
+
+inputs:
+
+  aws_region_name:
+    type: string
+    default: 'eu-west-1'
+
+  availability_zone:
+    type: string
+    default: 'eu-west-1b'
+
+  instance_type:
+    type: string
+    default: t3.medium
+
+  aws_image_name:
+    type: string
+    default: 'ami-0b850cf02cc00fdc8'
+
+  aws_subnet_id:
+    type: string
+    default: 'subnet-07896a88a91ea802f'
+
+  aws_vpc_id:
+    type: string
+    default: 'vpc-0d297ad8218931536'
+
+  aws_security_group_id:
+    type: string
+    default: 'sg-0bc113600cb5002a2'
+
+  agent_user:
+    description: >
+      The username of the agent running on the instance created from the image.
+    default: 'centos'
+
+  ssh_key_name:
+    type: string
+    default: 'cfy-agent-tests-key'
+
+  agent_key_name:
+    type: string
+    default: agent_key
+
+
+dsl_definitions:
+
+   client_config: &client_config
+    aws_access_key_id: { get_secret:  aws_access_key_id }
+    aws_secret_access_key: { get_secret:  aws_secret_access_key }
+    region_name: { get_input: aws_region_name }
+
+node_templates:
+
+  vm:
+    type: cloudify.nodes.aws.ec2.Instances
+    properties:
+      client_config: *client_config
+      agent_config:
+        install_method: none
+        user: { get_input: agent_user }
+        key: { get_attribute: [agent_key, private_key_export] }
+      resource_config:
+        ImageId: { get_input:  aws_image_name }
+        InstanceType: { get_input: instance_type }
+        kwargs:
+          Placement:
+            AvailabilityZone: { get_input:  availability_zone }
+          UserData: { get_attribute: [ cloud_init, cloud_config ] }
+          BlockDeviceMappings:
+          - DeviceName: '/dev/xvda'
+            Ebs:
+              VolumeSize: 22
```

Comment:
Do you need a big volume? 8Gb not enough?

Reply:
I'll try to run it with 8gb, might be enough

Reply:
executed well on 8gb